src/api/handlers/community.py

```diff
@@ -221,3 +221,21 @@ def super_admin_list(self, request):
     if err:
       return MassenergizeResponse(error=str(err), status=err.status)
     return MassenergizeResponse(data=communities)
+
+
+  # @admins_only
```

Comment:
ignore. will uncomment this before merge

Reply:
Done